tap_stripe/__init__.py

```diff
@@ -443,6 +446,11 @@ def sync_stream(stream_name):
         # when they are available via the API, so these streams will need
         # a short lookback window.
         if stream_name in IMMUTABLE_STREAMS:
+            if stream_name == "events":
+                # Start sync from 30 days before if bookmark/start_date is older than 30 days.
+                start_window = int(max(bookmark, (singer.utils.now() - timedelta(days=30)).timestamp()))
+                if start_window != bookmark:
+                    LOGGER.warning("Provided current bookmark/start_date is older than the last 30 days. So, starting sync for the last 30 days as Stripe Event API returns data for the last 30 days only.")
```

Comment:
@prijendev This line contains more than 120 characters. Did we make sure PEP8 standards are followed?
CC: @dbshah1212

Reply:
No, As this isn't refactoring release. 
It's quick bug fix release. 
Tap-Stripe is a candidate of refactoring so we can plan this at the time of refactoring for sure.